guardian/content-atom
# Add npm release

Following #162, adds a release for the corresponding npm package.

## How to test

I've run this workflow manually by running it with a hardcoded version number. You should be able to see the npm release [here.](https://www.npmjs.com/package/@guardian/content-atom-model/v/4.0.0-beta.0)
